public/components/common/seach/queryBar.tsx

```diff
@@ -9,28 +9,137 @@
  * GitHub history for details.
  */
 
-import React from 'react';
+import React, {Fragment, useEffect, useState, useCallback} from 'react';
 import {
+  EuiButtonEmpty,
   EuiFieldSearch,
   EuiFlexItem,
+  EuiSuggest,
+  EuiFlexGroup,
+  EuiSelectableTemplateSitewide,
+  EuiSelectableTemplateSitewideOption,
+  EuiSuggestItem,
+  EuiSuggestItemProps
 } from '@elastic/eui';
 import {
   IQueryBarProps
 } from './search';
 import {
   RAW_QUERY
 } from '../../../common/constants/explorer';
+import { Test } from 'mocha';
 
-export function QueryBar(props: IQueryBarProps) {
+const commands: EuiSuggestItemProps[] =[
+  {type: {iconType: 'search', color: 'tint1'} ,label: 'dedup'},
+  {type: {iconType: 'search', color: 'tint1'} ,label: 'eval'},
+  {type: {iconType: 'search', color: 'tint1'} ,label: 'fields'},
+  {type: {iconType: 'search', color: 'tint1'} ,label: 'rename'},
+  //{type: {iconType: 'search', color: 'tint1'} ,label: 'search'},
```

Comment:
Just curious, how come `search` as an option here is commented out and left in the `firstCommand` options while `source` is left in both?

Reply:
I think `search` and `source` can both be first commands, but `source` be second if `search` is the first command. I was still working on the grammar portion, so I just left source in there for now. Will prob remove it